devicetypes/smartthings/homeseer-multisensor.src/homeseer-multisensor.groovy

```diff
@@ -20,7 +20,7 @@ metadata {
 		capability "Sensor"
 		capability "Battery"
 
-		fingerprint deviceId: "0x2101", inClusters: "0x60,0x31,0x70,0x84,0x85,0x80,0x72,0x77,0x86"
+		fingerprint deviceId: "0x2101", inClusters: "0x60,0x31,0x70,0x84,0x85,0x80,0x72,0x77,0x86", deviceJoinName: "HomeSeer Multipurpose Sensor"
```

Comment:
Since this is a generic fingerprint I think "HomeSeer" should be left out.

Reply:
Brand name is also in DTH's name, so including DJN in this fingerprint has only device type naming unification purpose.